src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Spinner.Edit.cshtml

```diff
@@ -37,12 +44,24 @@
         <span class="hint">@settings.Hint</span>
     }
 </div>
-<script at="Foot">
-    $(function () {
-        $('#@(id)').spinner({
-            min: @min,
-            max: @max,
-            step: @step
-        });
-    });
+
+<script asp-name="numericFieldSpinner" at="Foot">
```

Comment:
I don't think name is allowed on a script tag.

Reply:
It's a server-side thing that does not get rendered to the client. Naming a script prevents the same script from rendering multiple times. If I remove this, the script is rendered for each spinner on the page. Naming it prevents this.

Reply:
ahhh I see LGTM then.